Source/WebKit/NetworkProcess/storage/OriginStorageManager.cpp

```diff
@@ -452,7 +452,7 @@ void OriginStorageManager::StorageBucket::deleteIDBStorageData(WallTime time)
 void OriginStorageManager::StorageBucket::deleteCacheStorageData(WallTime time)
 {
     if (m_cacheStorageManager)
-        m_cacheStorageManager = nullptr;
+        m_cacheStorageManager->close();
```

Comment:
I would not use close since it seems terminal, maybe reset?

Reply:
Changed to reset.